lib/menu.rb

```diff
@@ -0,0 +1,8 @@
+class Menu
+
+  attr_reader :dishes
+
+  def initialize
+    @dishes = { "Ramen": 3, "Burger": 2, "Coke": 5, "Chocolate": 3, "Parmigiano": 8}
```

Comment:
Metrics/LineLength: Line is too long. [84/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)